Codegen/test/resources/input/TypeCobol/CopyReplace4Colon/YxxxENT

```diff
@@ -0,0 +1,2 @@
+﻿           05  xxxENT::-FCT01.
```

Comment:
File without extension and with same content as YxxxENT.cpy

Reply:
I thought that you asked me to YxxxENT without extension!

Reply:
done 973c01a